src/sentry/conf/server.py

```diff
@@ -2732,12 +2726,7 @@ def build_cdc_postgres_init_db_volume(settings: Any) -> dict[str, dict[str, str]
     ),
     "clickhouse": lambda settings, options: (
         {
-            "image": "ghcr.io/getsentry/image-mirror-yandex-clickhouse-server:20.3.9.70"
-            if not APPLE_ARM64
-            # altinity provides clickhouse support to other companies
-            # Official support: https://github.com/ClickHouse/ClickHouse/issues/22222
-            # This image is build with this script https://gist.github.com/filimonov/5f9732909ff66d5d0a65b8283382590d
-            else "ghcr.io/getsentry/image-mirror-altinity-clickhouse-server:21.6.1.6734-testing-arm",
+            "image": "ghcr.io/getsentry/image-mirror-yandex-clickhouse-server:21.8.13.1",
```

Comment:
Im assuming you need the altinity stable at the end https://github.com/getsentry/image-mirror/pkgs/container/image-mirror-altinity-clickhouse-server/115571514?tag=21.8.13.1.altinitystable